Makefile

```diff
@@ -1,8 +1,8 @@
 # sourced by https://github.com/octomation/makefiles
 
 .DEFAULT_GOAL = test-with-coverage
-GIT_HOOKS     = post-merge pre-commit
-GO_VERSIONS   = 1.13 1.14
+GIT_HOOKS     = post-merge pre-commit pre-push
+GO_VERSIONS   = 1.13 1.14 1.15
```

Comment:
the right fix is `1.14 1.15`, without 1.13

https://github.com/octomation/go-tool/blob/master/Makefile#L5

Reply:
I recommend using force copy.